media-libs/tse3/tse3-0.3.1-r1.ebuild

```diff
@@ -17,17 +17,21 @@ IUSE="alsa oss"
 RDEPEND="alsa? ( media-libs/alsa-lib )"
 DEPEND="${RDEPEND}"
 
+PATCHES=(
+	"${FILESDIR}/${PN}-0.2.7-size_t-64bit.patch"
+	"${FILESDIR}/${PN}-0.2.7-gcc4.patch"
+	"${FILESDIR}/${P}-parallelmake.patch"
+	"${FILESDIR}/${P}+gcc-4.3.patch"
+)
+
 src_prepare() {
+	default
+	mv configure.in configure.ac || die "Moving configure.in -> .ac failed"
 	if use oss; then
-		cp "${WORKDIR}"/awe_voice.h src/
+		cp "${WORKDIR}"/awe_voice.h src/ || die "copy failed"
 		append-flags -DHAVE_AWE_VOICE_H
```

Comment:
try making this `append-cppflags` instead (this should be giving you a warning anyhow)